packages/help-center/src/components/help-center-article.tsx

```diff
@@ -37,10 +37,10 @@ export const HelpCenterArticle = () => {
 				if ( anchorId ) {
 					const element = document.getElementById( anchorId );
 					if ( element ) {
-						element.scrollIntoView();
+						element.scrollIntoView( { behavior: 'instant' } );
 					}
 				}
-			}, 0 );
+			}, 100 );
```

Comment:
This is added because I was experiencing erratic behaviour when opening the link on the Help Center due to some race condition:

![chrome-capture-2025-6-25](https://github.com/user-attachments/assets/d9c78ee4-7d1d-439e-b5f7-b7497efacfcc)

Related slack conversation: p1750847414906429-slack-C04DZ8M0GHW